Sources/Tests/UnitTests/Core/Settings.cs

```diff
@@ -58,5 +60,20 @@ public void AssertGetBackDespiteDispose()
             }
             Assert.Equal(67, MaxExpansionTermCount.Value);
         }
+        
+        [Theory]
+        [InlineData("x2=16")]
+        [InlineData("x2 + 4x +4 = 0")]
+        [InlineData("(x2+5)(x2+5)")]
+        [InlineData("3(x2+5)")]
+        [InlineData("integral(x3)")]
+        [InlineData("derivative(x2)")]
```

Comment:
Alright, let me help here. There is no point to test `x2`, because you already tested it. Now, you need to test `) 2` case. Like this:
```suggestion
        [InlineData("(a + 2) 2")]
        [InlineData("3 2")]
```
You can see, it also tests cases when power is added not to a variable, but to something else.

Reply:
Alright, thanks for the suggestions